### PR TITLE
Schema example changes

### PR DESCRIPTION
**Pull request by joshcarp**
see https://github.com/bufbuild/buf/issues/1432#issuecomment-1276726815

### CODE REVIEW

File: docs/reference/inputs.md [hunk 1]
```diff
@@ -71,25 +71,25 @@ $ buf build -o image.bin
 $ buf breaking --against image.bin
 ```
 
-## Specifying an input
+## Specifying a schema
```
Comment (ElliotMJackson) [inline docs/reference/inputs.md:74]: in this section “Buf schemas” are almost referred to like a first class citizen. like a "buf schema" is almost an entity deserving of a independent definition. Would it be better to remove the word "Buf" and obtain context in another way?

File: docs/reference/inputs.md [hunk 2]
```diff
@@ -71,25 +71,25 @@ $ buf build -o image.bin
...
 
-Buf inputs are specified as the first argument on the command line, and with the
-`--against` flag for the compare against input on `buf breaking`.
+Buf schemas are almost always specified as the first argument on the command line, and with the
```
Comment (ElliotMJackson) [inline docs/reference/inputs.md:76]:
```suggestion
Schemas are almost always specified as the first argument on the command line, and with the
```

Comment (ElliotMJackson) [reply docs/reference/inputs.md:76]: maybe adding before this...

```
In this context, "schemas" refer to...
```

Comment (ElliotMJackson) [reply docs/reference/inputs.md:76]: oh its in "terminology" - thats good enough.

File: docs/reference/inputs.md [hunk 3]
```diff
@@ -71,25 +71,25 @@ $ buf build -o image.bin
...
 
-The path specifies the path to the input. The options specify options to
-interpret the input at the path.
+The path specifies the path to the schema. The options specify options to
+interpret the schema at the path.
 
 ### format option
 
-The `format` option can be used on any input string to override the derived
+The `format` option can be used on any schema string to override the derived
```
Comment (joshcarp) [inline docs/reference/inputs.md:92]:
```suggestion
The `format` option can be used on any schema to override the derived
```

File: docs/reference/inputs.md [hunk 4]
```diff
@@ -6,11 +6,11 @@ title: Inputs
 The various I/O options for `buf` may seem a bit daunting, so we'll break down
 how this all fits together.
 
-In general, a Buf input is a collection of `.proto` files used by many of the
+In general, a Buf schema input is a collection of `.proto` files used by many of the
```
Comment (joshcarp) [inline docs/reference/inputs.md:9]:
```suggestion
In general, a schema input for `buf` is a collection of `.proto` files used by many of the
```

File: docs/reference/inputs.md [hunk 5]
```diff
@@ -71,25 +71,25 @@ $ buf build -o image.bin
...
 
-The path specifies the path to the input. The options specify options to
-interpret the input at the path.
+The path specifies the path to the schema. The options specify options to
+interpret the schema at the path.
 
 ### format option
 
-The `format` option can be used on any input string to override the derived
+The `format` option can be used on any schema to override the derived
```
Comment (joshcarp) [inline docs/reference/inputs.md:92]:
```suggestion
The `format` option can be used on any flag or argument to override the derived
```

File: docs/reference/inputs.md [hunk 6]
```diff
@@ -71,25 +71,25 @@ $ buf build -o image.bin
...
+specified as the first argument. Schemas are specified as a string and have this
 structure:
 
 ```
 path#option_key1=option_value1,option_key2=option_value2
 ```
 
-The path specifies the path to the input. The options specify options to
-interpret the input at the path.
+The path specifies the path to the schema. The options specify options to
```
Comment (joshcarp) [inline docs/reference/inputs.md:87]:
```suggestion
The path specifies the path of the input flag or argument. The options specify options to
```

**Pull request closed without merging**